taqueria/.taq/state.json

```diff
@@ -1,16 +1,24 @@
 // WARNING: This file is autogenerated and should NOT be modified
 {
```

Comment:
Do we need this file in the merge a well?

Reply:
Ya this should be in .gitignore